src/main/io/osd.c

```diff
@@ -916,7 +922,9 @@ static bool osdDrawSingleElement(uint8_t item)
         {
             int32_t alt = osdGetAltitude();
             osdFormatAltitudeSymbol(buff, alt);
-            if ((osdConvertDistanceToUnit(alt) / 100) >= osdConfig()->alt_alarm) {
+            uint16_t neg_alt_alarm = osdConfig()->neg_alt_alarm;
+            if ((osdConvertDistanceToUnit(alt) / 100) >= osdConfig()->alt_alarm ||
```

Comment:
This still converts between metric/imperial. Isnt it inconsistent to use different units for some alarms and use only SI units for other alarms?

Reply:
It’s converting for the already existing altitude alarm. Didn’t want to change that without asking first. See #2297. If you agree, I can push another commit on top of this PR changing alt_alarm too.